skrifa/src/patchmap.rs

```diff
@@ -0,0 +1,393 @@
+//! Loads incremental font transfer <https://w3c.github.io/IFT/Overview.html> patch mappings.
```

Comment:
Given that this is somewhat low level and we've started adding `std` dependent code in `read-fonts`, I think this should be moved there under the `ift` module and behind a `std` feature gate.

The only complication I can see is the use of the `Charmap` selection and iterator logic but my preference would be to move that to read-fonts as well (it'll be useful for shaping anyway).

Reply:
Since Charmap hasn't been moved yet I'll leave this here for this PR and move it into read-fonts after Charmap is moved.

Reply:
Sounds good. Let’s land this and I’ll ping you when the necessary functionality is available in read-fonts